kaggle_environments/envs/lux_ai_2022/luxai2022/replay/replay.py

```diff
@@ -0,0 +1,27 @@
+import os
+from typing import List, Tuple
+from luxai2022.state import State
+import os.path as osp
+import json
+def decode_replay_file(replay_file) -> Tuple[State, List]:
+    """
+    Takes an input replay file of any kind from any source and extracts the full trajectory with observations
+    or just initial state and actions if observations aren't there
+    """
+    ext = osp.splitext(replay_file)[-1]
+    if ext == ".json":
+        # probably kaggle replay
+        with open(replay_file, "r") as f:
+            replay = json.load(f)
+        # TODO - check actual key
+        init_state = replay["init_state"]
```

Comment:
`["steps"][0]` maybe?

Reply:
oh i think we don't use this code anymore, will remove

Reply:
This is for something in the future actually for decoding compressed replays into full observation replays (mostly for data-based solutions)